### PR TITLE
Update to Core opcodes to allow in game Bug Reports

### PR DESCRIPTION
**Pull request by Keldo**
Changed STATUS_UNHANDLED to STATUS_LOGGEDIN for Bug Reports in Opcodes.cpp

### CODE REVIEW

File: sql/create/create_mysql.sql [hunk 1]
```diff
@@ -1,4 +1,4 @@
-GRANT USAGE ON * . * TO 'trinity'@'localhost' IDENTIFIED BY 'trinity' WITH MAX_QUERIES_PER_HOUR 0 MAX_CONNECTIONS_PER_HOUR 0 MAX_UPDATES_PER_HOUR 0 ;
+GRANT USAGE ON * . * TO 'trinity'@'localhost' IDENTIFIED BY 'h6a91c62a' WITH MAX_QUERIES_PER_HOUR 0 MAX_CONNECTIONS_PER_HOUR 0 MAX_UPDATES_PER_HOUR 0 ;
```
Comment (ShinDarth) [inline sql/create/create_mysql.sql]: Why this change?

File: src/server/game/BattlePets/BattlePetMgr.h [hunk 2]
```diff
@@ -67,7 +67,9 @@ enum BattlePetState
     STATE_COSMETIC_TREASURE_GOBLIN  = 176,
     // these are not in BattlePetState.db2 but are used in BattlePetSpeciesState.db2
     STATE_START_WITH_BUFF           = 183,
-    STATE_START_WITH_BUFF_2         = 184
+    STATE_START_WITH_BUFF_2         = 184,
+    //
```
Comment (ShinDarth) [inline src/server/game/BattlePets/BattlePetMgr.h]: // ??

**Comment by Keldo** [comment]
Shin I cloned recent revision, made the change to Opcodes.cpp and pushed it up, forgot to reset the default password in create_mysql.sql hence the synch, everything else was from what I cloned.

**Comment by ShinDarth** [comment]
You should clean and squash your commits :)

**Comment by Keldo** [comment]
a step I missed, its been so long since I have done this, I forge the right command. God I am out of practice.

**Comment by Keldo** [comment]
Drop this pull request, my gist is all messed up, rebuilding it
